### DISCUSSION TITLE
Bed and Nozzle dont meet temperature

### DISCUSSION THREAD

**Discussion by rizeddd** [Jyers/Marlin#202] [Q&A] [answered]
Hi , i've downloaded the version to edit , i put everything together for my build but im getting only 45c at the bed and 190 at the nozzle , what can this be ? I've tried to check everything and didnt come to an conclusion 

Thanks

**Comment by rizeddd** [comment]
* i've rolled back to the previously installed fw and bed and nozzle works, but i really want this to work

**Comment by rizeddd** [comment]
I leave here the file i've made if any kind soul can get the error 
Thanks


[Marlin-Ender_3_V2_Rewrite.zip](https://github.com/Jyers/Marlin/files/6097857/Marlin-Ender_3_V2_Rewrite.zip)

**Comment by Jyers** [comment]
Just to make sure I understand correctly. With this firmware, your bed is only able to go up to 45 and your hotend will only go to 190?

**Reply by rizeddd** [reply]
Hi , yes thats it ! 
thanks

**Reply by Jyers** [reply]
If you could give me the exact procedure you are using to heat them up and then exactly how they are behaving (ie target temp and current temp) that might help diagnose your issue.

**Comment by Jyers** [comment, marked as answer]
Any update on this?

**Reply by rizeddd** [reply]
Hi , sorry i changed to your Extensible UI version i was using the Rewrite one and that fixed it , but now i found another problem for me , when i want to extrude via screen , the motor spins too fast , what can it be ? 

Thanks

**Reply by Jyers** [reply]
I switched to the marlin default extrude feedrate for manual moves, if it is causing your extruder to skip, I would recommend either increasing your temp or checking for a clog.

**Reply by rizeddd** [reply]
Hi , thanks for your time , is there anyway i can change it to be slower on Marlin ?

**Reply by Jyers** [reply]
The setting for it can be found in configuration_adv.h "#define MANUAL_FEEDRATE"
